src/pages/beneficiary/registerBeneficiary/BeneficiaryRegistrationForm.tsx

```diff
@@ -3,6 +3,14 @@ import { TextField, Typography, Button, Grid } from "@mui/material";
 import { useDispatch } from "react-redux";
 import { createBeneficiary } from "../../../store/slices/beneficiarySlice";
 import { AppDispatch } from "../../../store";
+import styled from "styled-components";
+
+const StyledTextField = styled(TextField)`
+  .MuiInputBase-root {
+    background-color: #d58855;
+    color: white;
+  }
+`;
 
 const formFields = [
```

Comment:
I notice you using this format to generate form fields here, but not on the Login or Register pages. This doesn't have to happen right now, but please add a new task for later to refactor all forms to use the same setup/